server/middlewares/auth.js

```diff
@@ -1,6 +1,45 @@
-const auth = (req, res, next) => {
-    // placeholder auth middleware
-    return next();
+require("dotenv").config();
+const { verify_jwt } = require("./verify_jwt");
+const axios = require("axios");
+
+const auth = async (req, res, next) => {
+    // make it possible to skip actual auth for testing/development
+    var skip = process.env.SKIP_AUTH === "true";
+    if (skip) {
+        return next();
+    }
+
+    const access_token = req.header("accessToken");
+    const refresh_token = req.header("refreshToken");
+
+    if (!access_token || !refresh_token) {
+        return res.status(403).send("missing token");
```

Comment:
403 Forbidden ist in diesem Fall nicht richtig, da kein Autorisierungsversuch durchgeführt wurde. 401 Unauthorized wäre richtig.

Reply:
fixed